flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/tasks/StreamTaskITCase.java

```diff
@@ -110,6 +118,47 @@ public void testFailInEndOfConstructor() throws Exception {
                                 .build(EXECUTOR_RESOURCE.getExecutor()));
     }
 
+    @Test
+    public void streamTaskAsyncExceptionHandlerHandleExceptionInInitializingDuration()
```

Comment:
There are some issues about the method name:
1. Test method name should start with `test`.
2. It's too long, and the Class Name is about `StreamTask`, so I think the `StreamTask` can be removed for the method name.
3. Rename the `Initializing` to `restoring`

How about `testAsyncExceptionCanBeHandledDuringRestoring`?

Reply:
> How about `testAsyncExceptionCanBeHandledDuringRestoring`?

Thanks for the items, which is fine to improve the code style.
updated.